vllm_spyre/platform.py

```diff
@@ -19,13 +19,13 @@
 
 class SpyrePlatform(Platform):
     _enum = PlatformEnum.OOT
-    device_name: str = "spyre"
+    device_name: str = "cpu"
     device_type: str = "cpu"
     supported_quantization: list[str] = ["gptq"]
 
     @classmethod
     def get_device_name(cls, device_id: int = 0) -> str:
-        return "spyre"
+        return "cpu"
```

Comment:
I guess technically returning `spyre` here is still safe since the offending vllm code uses `current_platform.device_name` directly.

I think this method is meant for returning the human readable device name, based on this code snippet:
```
        try:
            from vllm.platforms import current_platform
            device_name = current_platform.get_device_name().lower()
        except Exception:
            # This is only used to set default_max_num_batched_tokens
            device_name = "no-device"

        if "h100" in device_name or "h200" in device_name:
```

Which is for sure a bit confusing but if that's the case then I think we should keep `spyre` here for the humans